component/pom.xml

```diff
@@ -141,6 +163,19 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
It is already called in the parent pom and can be removed

Reply:
Fixed with commit: e4ab249